packages/ses/index.js

```diff
@@ -15,3 +15,4 @@
 import './src/lockdown-shim.js';
 import './src/compartment-shim.js';
 import './src/assert-shim.js';
+import './src/console-shim.js';
```

Comment:
We may eventually need an entry in `package.json` and a top-level `./console-shim.js`, but not today.

Reply:
Why? By "entry in `package.json`, do you mean in `"exports":`? If so, I'm surprised. I thought that's what you just talked me out of. Am I confused?

Reply:
Talked to @kriskowal . This is fine. Resolving.